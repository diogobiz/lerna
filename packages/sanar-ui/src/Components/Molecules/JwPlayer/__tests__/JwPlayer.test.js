import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESJwPlayer from '../'

it('renders correctly', () => {
    const component = (
        <ESJwPlayer
            playerId='playerId'
            playlist='https://cdn.jwplayer.com/v2/media/yp34SRmf'
            playerScript='https://cdn.jwplayer.com/libraries/jX7FSJdG.js'
        />
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
