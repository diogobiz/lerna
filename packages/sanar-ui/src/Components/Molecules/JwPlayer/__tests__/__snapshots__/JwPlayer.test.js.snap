// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="es-jw-player"
>
  <div
    className="ant-spin ant-spin-spinning es-spin es-spin__dark es-jw-player__loader"
    style={
      Object {
        "height": "undefinedpx",
      }
    }
  >
    <span
      className="ant-spin-dot ant-spin-dot-spin"
    >
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
    </span>
  </div>
  <div>
    <div
      id="playerId"
    />
  </div>
</div>
`;
